packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -113,6 +114,9 @@ export interface TableListViewTableProps<
   refreshListBouncer?: boolean;
   onFetchSuccess: () => void;
   setPageDataTestSubject: (subject: string) => void;
+
+  /** resolve user profiles for the user filter and creator functionality */
+  bulkGetUserProfiles?: (uids: string[]) => Promise<UserProfile[]>;
```

Comment:
Hum, actually looking at the PR, it seems the only API you're using is `userProfile.bulkGet`.

It probably wasn't merged when you started the PR, but the `userProfile` API is now available from Core:

https://github.com/elastic/kibana/blob/0f3ecf739e2040a8831c508f1df92c11bb5028dc/packages/core/user-profile/core-user-profile-browser/src/service.ts#L34-L44

Accessible via `CoreStart.userProfile`.

Would you mind adapting the PR to use the API exposed from Core instead of the one exposed from the `security` plugin?

Reply:
Sounds great. Will do!